components/raftstore/src/store/config.rs

```diff
@@ -425,6 +424,14 @@ impl Config {
             ));
         }
 
+        // The adjustment of this value is related to the number of regions, usually 4096 is
+        // already a large enough value
+        if self.raft_max_inflight_msgs == 0 || self.raft_max_inflight_msgs > 4096 {
```

Comment:
As I remember, some users are using a value bigger than 4096, maybe we need to enlarge the upper bound.

Reply:
Fixed, thanks!